test/integration/unregistered_user_can_view_categories_of_items_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+class UnregisteredUserCanViewCategoriesOfItemsTest < ActionDispatch::IntegrationTest
+
+  test ' user can view categories linked ' do
+  category_one = Category.create(name:'coffee')
+      category_one.items.create(title: "pour over", description: 'it taste really good', price: 4.00, image: "google.com")
+      category_one.items.create(title: "drip",      description: 'it taste good',         price: 4.00, image: "google.com")
```

Comment:
Line is too long. [123/80]<br>Unnecessary spacing detected.<br>Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.